docs/helm/examples/with-ai.md

```diff
@@ -23,9 +23,9 @@ helm install acs alfresco/alfresco-content-services \
 --set externalPort="443" \
 --set externalProtocol="https" \
 --set externalHost="acs.YOUR-DOMAIN-NAME" \
---set persistence.enabled=true \
---set persistence.storageClass.enabled=true \
---set persistence.storageClass.name="nfs-client" \
+--set repository.persistence.enabled=false \
```

Comment:
why false? if it's a required we could briefly explain it because I wasn't expecting that